tests/Integrations/PDO/PDOTest.php

```diff
@@ -16,7 +16,8 @@ final class PDOTest extends IntegrationTestCase
     public static function setUpBeforeClass()
     {
         parent::setUpBeforeClass();
-        IntegrationsLoader::load();
+        putenv('DD_TRACE_SANDBOX_ENABLED=false');
```

Comment:
I think ideally when porting integrations to sandboxing we should be using the exact same test suite for new and old implementation.

I don't mind refactoring - I think the new testsuite looks really great, but if possible both should share as much code as possible. Sometimes new tests or minute differences can slip with two separate test cases which might increase a chance of a regression.

WDYT about using the same test suite for both variants ? Simple inheritance could be used to make two identical sets.